code/modules/events/anomaly_cola.dm

```diff
@@ -0,0 +1,52 @@
+#define COLA_AMT 5
+
+/datum/round_event_control/infcola
+	name = "Cola Infinite"
+	typepath = /datum/round_event/infcola
+	min_players = 1
+	max_occurrences = 10
+	can_malf_fake_alert = TRUE
+
+/datum/round_event/infcola
+	announceWhen = 2
+
+/datum/round_event/infcola/announce(fake)
+	priority_announce("Our long-range anomaly scanners have detected leakage from a soda filled dimension. Nanotrasen is not responsible for any damages caused by these anomalous canisters.", "General Alert", SSstation.announcer.get_rand_alert_sound())
```

Comment:
A soda filled dimension? This needs to me made to align with lore / not be memey.

Reply:
ok what about space cola co experimenting with bluespace tech and causing this